packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -157,7 +157,8 @@ class IOSDevice extends Device {
     bool clearLogs: false,
     bool startPaused: false,
     int debugPort: observatoryDefaultPort,
-    Map<String, dynamic> platformArgs
+    Map<String, dynamic> platformArgs,
+    String dartFlags,
```

Comment:
I think the travis errors are just the extraneous comma on the `String dartFlags,` line here.